builder/proxmox/common/step_start_vm.go

```diff
@@ -43,9 +43,7 @@ func (s *stepStartVM) Run(ctx context.Context, state multistep.StateBag) multist
 		Name:         c.VMName,
 		Agent:        agent,
 		QemuKVM:      kvm,
-		Boot:         "cdn", // Boot priority, c:CDROM -> d:Disk -> n:Network
 		QemuCpu:      c.CPUType,
-		Description:  "Packer ephemeral build VM",
```

Comment:
This should be automatically either removed or replaced with the configured template description as part of the template-conversion step. Is this not happening for you?
I believe it is a good thing to have this in place during the build so others seeing the VM can understand what it is.

Reply:
I tried again today and I can confirm that the `description` parameter is passed to the final VM. I've the following template.

```
$ qm config 200
name: ubuntu-20.04
description: Packer ephemeral build VM
template: 1
[...]
```

I've created a VM from this template.

```
$ qm config 108
name: templated
description: Packer ephemeral build VM
[...]
```
This is not critical but if the description is passed to the final VM, it's not that useful.

Reply:
What do you think of this change @carlpett ? Should we go on with it ? 🙂  cheers !

Reply:
Ooops, must have missed this notification in the christmas mail heaps. Sorry!
What I would have expected in this is that the template with VMID 200 shouldn't have a description at all, it's supposed to be removed before being converted to a template. If it has stuck around somehow, then it's expected that it'll get cloned onto VMs built from the template.

I'm unable to reproduce this locally, though. Could you tell me what version of Packer and Proxmox you have, and a sample build configuration?

Reply:
bump @aerialls -- did you see Carl's comment above?